assets/app/scripts/filters/resources.js

```diff
@@ -380,4 +380,13 @@ angular.module('openshiftConsole')
 
       return false;
     };
+  })
+  .filter('projectOverviewURL', function(Navigate) {
```

Comment:
I'm also fine with it taking a project or project name (if `p.metadata && p.metadata.name` exists, use that, etc)